databricks/koalas/indexes/base.py

```diff
@@ -110,6 +110,31 @@ def __new__(cls, data=None, dtype=None, copy=False, name=None, tupleize_cols=Tru
         if not is_hashable(name):
             raise TypeError("Index.name must be a hashable type")
 
+        if isinstance(data, Series):
```

Comment:
I'm wondering if it's worth adding docstrings example when `data` is Series or Index.

Reply:
sounds good.